docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/migrate-to-hybrid-cloud-scenarios.md

```diff
@@ -60,5 +60,5 @@ Microsoft Azure Stack Development Kit is a single-node deployment of Azure Stack
     <https://azure.microsoft.com/pricing/hybrid-benefit/>
 
 >[!div class="step-by-step"]
->[Previous](modernize-your-apps-lifecycle-with-ci-cd-pipelines-and-devops-tools-in-the-cloud.md)
+>[Previous](modernize-ci-cd-pipelines-devops-tools.md)
```

Comment:
```suggestion
>[Previous](life-cycle-ci-cd-pipelines-devops-tools)
```

Reply:
@mairaw this suggestion removes the `.md` extension, but the same suggestion for the "Next" link doesn't remove the extension.

Reply:
Yeah it was my bad. Didn't have coffee yet at that time. 😄